.sequelizerc

```diff
@@ -1,7 +1,7 @@
 const path  = require('path')
 
 module.exports = {
-    "config": path.resolve('./build/server/database/config', 'config.js'),
+    "config": path.resolve('./build/server/database/config', 'config.json'),
```

Comment:
Just curious, should this remain `config.js`? Didn't notice anything regarding the change of configs on the PR overview page. 👀

Reply:
Good catch @aniruddha-adhikary! @rlyss this should be `config.js`.

Reply:
Yup thanks for the reminder! I changed it to test out some stuff in sqlite and was going to restore it back to js in the next commit.